includes/core/server/request/class-create-intent.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Class file for WCPay\Core\Server\Request\Create_Intent.
+ *
+ * @package WooCommerce Payments
+ */
+
+namespace WCPay\Core\Server\Request;
+
+use WCPay\Core\Exceptions\Invalid_Request_Parameter_Exception;
+use WCPay\Core\Server\Request;
+use WC_Payments_API_Client;
+
+/**
+ * Request class for creating intents.
+ */
+class Create_Intent extends Request {
```

Comment:
Nothing to resolve here, just noting that this class was used as a dummy during development and is currently not representing the actual request. I would have suggested deleting it if there were no tests already written for it.

Let's keep that in mind when we start replacing requests throughout the gateway.

Reply:
Noted.